tests/sentry/api/endpoints/test_sentry_apps.py

```diff
@@ -76,6 +75,37 @@ def test_users_see_published_apps(self):
             }
         } in json.loads(response.content)
 
+    def test_superuser_can_filter_apps_on_status(self):
```

Comment:
I think the filtering is for more than just superusers, right? We should have tests for filtering on status for normal users too.